src/server/config/auth.js

```diff
@@ -0,0 +1,37 @@
+const passport = require('passport');
+const GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+
+const { ENDPOINT_URL } = require('../common/variables');
+
+// Use GoogleStrategy to authenicate user
+passport.use(
+  new GoogleStrategy(
+    {
+      clientID:
+        '535360565912-hvhhi6g8j4ot9l7maikebnkv8v1h8rgb.apps.googleusercontent.com',
```

Comment:
Please don't commit API ID's and secrets - consider moving them to the ENV variables